ihatemoney/utils.py

```diff
@@ -150,6 +150,26 @@ def list_of_dicts2csv(dict_to_convert):
     return csv_file
 
 
+def csv2list_of_dicts(csv_to_convert):
+    """Take a csv in-memory file and turns it into
+    a list of dictionnaries
+    """
+    csv_file = TextIOWrapper(csv_to_convert, encoding="utf-8")
+    reader = csv.DictReader(csv_file)
+    result = []
+    for r in reader:
+        # cospend filtering
```

Comment:
Could you please expand here on why this is required and what's it's doing? Thanks :-)

Reply:
Yup
by the way I thought about also exporting data in cospend format (with lines for users and a table for categories)
it would simplify our csv import code AND render IHM exports compatible with cospend